### PR TITLE
Only run Linux fuchsia_ctl if fuchsia_ctl is modified

### PR DESCRIPTION
**Pull request by dnfield**
The fuchsia_ctl test is relatively expensive and takes a while to run.

### CODE REVIEW

**Comment by dnfield** [comment]
@akbiggs FYI

File: .ci.yaml [hunk 1]
```diff
@@ -81,6 +81,8 @@ targets:
     timeout: 90
     properties:
       add_recipes_cq: "true"
+    runIf:
+      - packages/fuchsia_ctl/
```
Comment (CaseyHillers) [inline .ci.yaml:85]: You probably also want to add https://github.com/flutter/packages/tree/main/.ci and .ci.yaml as triggers as well

Comment (dnfield) [reply .ci.yaml:85]: Good call, did those and `script/`.

**Review by CaseyHillers — APPROVED** [review]

**Review by stuartmorgan — APPROVED** [review]
Nice! Thanks for this.

**Pull request merged**